frontend/packages/pipelines-plugin/src/components/import/pipeline/webhook-utils.tsx

```diff
@@ -0,0 +1,38 @@
+import * as React from 'react';
+import { AlertVariant } from '@patternfly/react-core/dist/esm/components/Alert';
+import i18next from 'i18next';
+import { RouteLinkAndCopy } from '@console/internal/components/routes';
+import { K8sResourceKind, RouteKind } from '@console/internal/module/k8s';
+import { EventListenerModel } from '@console/pipelines-plugin/src/models';
+import { getRouteByName } from './pipeline-template-utils';
+
+export const getWebhookToastMessage = (route: RouteKind) => ({
+  variant: AlertVariant.info,
+  title: i18next.t('pipelines-plugin~Webhook added'),
+  content: (
+    <>
+      {i18next.t(`pipelines-plugin~Webhook created successfully.`)}
+      <p>
+        <RouteLinkAndCopy route={route} />
+      </p>
+    </>
+  ),
+  timeout: true,
+  dismissible: true,
+});
+
+export const addWebhookToast = async (resources: K8sResourceKind[], toastContext) => {
```

Comment:
should we add type toastContextType

Reply:
This file is removed, as we dont need toast at the moment (discussed with UX on this).